src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Changed.cs

```diff
@@ -118,4 +118,128 @@ public static void FileSystemWatcher_Changed_FileInNestedDirectory()
+            {
+                // Attach the FSW to the existing structure
+                watcher.Path = Path.GetFullPath(dir.Path);
+                watcher.Filter = "*";
+                watcher.NotifyFilter = NotifyFilters.Attributes | NotifyFilters.CreationTime | NotifyFilters.DirectoryName | NotifyFilters.FileName | NotifyFilters.LastAccess | NotifyFilters.LastWrite | NotifyFilters.Security | NotifyFilters.Size;
+                watcher.EnableRaisingEvents = true;
+
+                // Change the nested file and verify we get the changed event
+                byte[] bt = new byte[8000];
+                file.Write(bt, 0, bt.Length);
+                file.Flush();
+            }
+
+            Utility.ExpectEvent(eventOccured, "file changed");
+        }
+    }
+
+    [Fact]
+    public static void FileSystemWatcher_Changed_PreSeededNestedStructure()
+    {
+        // Make a nested structure before the FSW is setup
+        using (var dir = Utility.CreateTestDirectory())
+        using (var watcher = new FileSystemWatcher())
+        using (var dir1 = Utility.CreateTestDirectory(Path.Combine(dir.Path, "dir1")))
+        using (var dir2 = Utility.CreateTestDirectory(Path.Combine(dir1.Path, "dir2")))
+        using (var dir3 = Utility.CreateTestDirectory(Path.Combine(dir2.Path, "dir3")))
+        {
+            AutoResetEvent eventOccured = Utility.WatchForEvents(watcher, WatcherChangeTypes.Changed);
+            byte[] bt = new byte[4096];
+
+            using (FileStream file = File.Create(Path.Combine(dir3.Path, "testfile.txt")))
+            {
+                // Attach the FSW to the existing structure
+                watcher.Path = Path.GetFullPath(dir.Path);
+                watcher.Filter = "*";
+                watcher.NotifyFilter = NotifyFilters.Attributes | NotifyFilters.CreationTime | NotifyFilters.DirectoryName | NotifyFilters.FileName | NotifyFilters.LastAccess | NotifyFilters.LastWrite | NotifyFilters.Security | NotifyFilters.Size;
+                watcher.IncludeSubdirectories = true;
+                watcher.EnableRaisingEvents = true;
+
+                // Change the nested file and verify we get the changed event
+                file.Write(bt, 0, bt.Length);
+                file.Flush();
+            }
+
+            Utility.ExpectEvent(eventOccured, "file changed", 10000);
+
+            // Stop the FSW
+            watcher.EnableRaisingEvents = false;
+
+            using (FileStream file2 = File.Create(Path.Combine(dir3.Path, "testfile2.txt")))
+            {
+                // Start the FSW, write to the file, expect to pick up the write
+                watcher.EnableRaisingEvents = true;
+                file2.Write(bt, 0, bt.Length);
+                file2.Flush();
+            }
+
+            Utility.ExpectEvent(eventOccured, "second file change");
+        }
+    }
+
+    [Fact, ActiveIssue("1477", PlatformID.Windows)]
```

Comment:
An overload of ActiveIssue's ctor accepts Int32, so you shouldn't need the quotes around the issue number.